src/pages/send/multi-send/multi-send.ts

```diff
@@ -264,20 +264,20 @@ export class MultiSendPage extends WalletTabsChild {
   }
 
   private checkCoinAndNetwork(data): boolean {
-    let isValid;
-
-    isValid = this.addressProvider.checkCoinAndNetworkFromAddr(
-      this.wallet.coin,
-      this.wallet.network,
-      data
+    const addrData = this.addressProvider.getCoinAndNetwork(
+      data,
+      this.wallet.network
     );
+    let isValid =
+      this.wallet.coin == addrData.coin && addrData.coin == this.wallet.network;
```

Comment:
the send part should be `addrData.network`

Reply:
thanks! fixed